Assets/Scripts/AWSPollyGetter.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Amazon.Polly;
+using Amazon.Polly.Model;
+using System.IO;
+using System;
+using System.Text;
+
+namespace MoveToCode {
+    public class AWSPollyGetter : Singleton<AWSPollyGetter> {
+        private const string APIPath = "Assets/Resources/PollyAPI1.txt";
+
+        private const string speakstyle = "pitch=\"x-high\" rate=\"100%\"";
+        private VoiceId speakvoice = VoiceId.Joanna;
+
+        private AmazonPollyClient apc;
+        public bool isWorking = true;
+        private bool onlineState;
+
+        public AWSPollyGetter(bool onlineState) {
+            this.onlineState = onlineState;
+            if (onlineState) { 
+                StreamReader reader = new StreamReader(APIPath);
+                try {
+                    apc = new AmazonPollyClient(reader.ReadLine(), reader.ReadLine(), Amazon.RegionEndpoint.USWest1);

```

Comment:
can you move the region out to the config file with the aws config (aka where you put your api keys)

Reply:
it's a text file, so i'll have to make a dictionary